src/rust/macros/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "macros"
+version = "0.1.0"
+authors = ["Sean Lynch <seanl@twitter.com>"]
+edition = "2018"
+license = "Apache-2.0"
```

Comment:
probably worth putting a description in there too. I'm wondering if we want to put this as rust/utils or something. But calling it 'macros' for now seems fine.